### PR TITLE
BHBC-1887, 1888: Add environment variables to platform

### PR DESCRIPTION
**Pull request by curtisupshall**
# Overview

 - New env var to specify elastic search EML index
 - New env var to specify S3 path prefix

## Links to Jira tickets

https://quartech.atlassian.net/browse/BHBC-1887
https://quartech.atlassian.net/browse/BHBC-1888

## Description of relevant changes
- Add a new env var, for local and openshift builds, that specifies the name of the elastic search index for the EML data.
  - If not provided, default to the current value: `"eml"`.
- Add a new env var, for local and openshift builds, that specifies an optional prefix for S3 file paths
  - If not provided, default to the current value: `"platform"`.

### CODE REVIEW

File: Containers/backup/templates/backup/backup-deploy.yaml [hunk 1]
```diff
@@ -477,3 +482,6 @@ parameters:
   - name: OBJECT_STORE_BUCKET_NAME
     displayName: Bucket name
     description: Object Store Bucket Name
+  - name: S3_KEY_PREFIX
+    displayName: S3 Key Prefix
+    description: Optional prefix for S3 file keys
```
Comment (curtisupshall) [inline Containers/backup/templates/backup/backup-deploy.yaml]: Might mention that the default value is "platform" 🤔

**Review by anissa-agahchen — DISMISSED** [review]

**Review by NickPhura — DISMISSED** [review]
Nice work

File: api/src/utils/file-utils.ts [hunk 2]
```diff
@@ -139,7 +140,7 @@ export interface IS3FileKey {
  * @return {*}  {string}
  */
 export function generateS3FileKey(options: IS3FileKey): string {
-  const keyParts: (string | number)[] = ['platform'];
+  const keyParts: (string | number)[] = [S3_KEY_PREFIX];
 
   if (options.submissionId) {
     keyParts.push(options.submissionId);
```
Comment (NickPhura) [inline api/src/utils/file-utils.ts:147]:
```suggestion
    keyParts.push('submissions');
    keyParts.push(options.submissionId);
```

Sorry one extra change while we're editing this function.

Comment (NickPhura) [reply api/src/utils/file-utils.ts:147]: We have this pattern in sims, but didn't get carried over for w/e reason.  Will help add some clarity to the path in S3.

**Review by NickPhura — APPROVED** [review]

**Pull request merged**